modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/VolatilePageMemoryTableStorage.java

```diff
@@ -174,7 +178,11 @@ private VersionChainTree createVersionChainTree(int partId) throws StorageExcept
     CompletableFuture<Void> clearStorageAndUpdateDataStructures(AbstractPageMemoryMvPartitionStorage mvPartitionStorage) {
         VolatilePageMemoryMvPartitionStorage volatilePartitionStorage = (VolatilePageMemoryMvPartitionStorage) mvPartitionStorage;
 
-        volatilePartitionStorage.destroyStructures();
+        volatilePartitionStorage.destroyStructures().whenComplete((res, ex) -> {
+            if (ex != null) {
+                LOG.error("Could not destroy structures", ex);
```

Comment:
Should there be any extra information here? Like `tableId` or something similar?